### ISSUE TITLE
CodeMap Explorer for all Opened Files

### ISSUE THREAD

**Issue by EmersonSuguimoto** [oleg-shilo/codemap.vscode#90]
Is it possible to have CodeMap treeview for all opened files? Not only for the current selected file.

**Comment by oleg-shilo** [comment]
Not really. I cannot imagine an adequate UX for that. IE IDE has five files open but only one is visible. So having five CodeMap views available does not make sense. You still see only a single document at the time.

VSCode provides a placeholder for custom views. One pwr window. Thus it is one custom view (e.g. CodeMap) per window.

**Comment by EmersonSuguimoto** [comment]
I used to work with Sourcecookifier in Notepad++, It maps every customized filter in regex in a single TreeView, which makes a lot easy to navigate through multiple files. It has also the "Go To Definition" functionality embedded that helps the navigation.
Could this be implemented?

**Comment by oleg-shilo** [comment]
I created a few extensions for Notepad++, including CS-Script that converts Notepad++ into an IDE for C# and in fact, it also integrates a CodeMap equivalent functionality. 

>to have CodeMap treeview for all opened files? Not only for the current selected file.

Notepad follows the very same usability pattern as VSCode and VS and I say all these IDEs always show the document metadata view for the "currently selected file". IE Document Outline window in VS.

Thus, most likely we are not talking about the same things. 
Can you please explain the UX you are trying to achieve? Then I will be able to tell you if it can be done or not.
